Terraform_modules/web_srv.tf

```diff
@@ -0,0 +1,25 @@
+data "terraform_remote_state" "vpc" {
+  backend = "local"
+
+  config = {
```

Comment:
state should be in s3

Reply:
> state should be in s3

already update everything, review please